.github/workflows/build.yml

```diff
@@ -13,7 +13,11 @@
 # limitations under the License.
 name: build
 
-on: ['push']
+on:
+  push:
+    paths-ignore:
+    - 'docs/**'
+    - 'deploy/helm/**'
```

Comment:
The same change needs to be done for pull request workflow also. Can you also add, changelog directory to `paths-ignore`